src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/Touch/TouchEffect.shared.cs

```diff
@@ -1160,10 +1160,20 @@ internal void RaiseHoverStatusChanged()
 			=> weakEventManager.RaiseEvent(Element, new HoverStatusChangedEventArgs(HoverStatus), nameof(HoverStatusChanged));
 
 		internal void RaiseCompleted()
-			=> weakEventManager.RaiseEvent(Element, new TouchCompletedEventArgs(CommandParameter), nameof(Completed));
+		{
+			var element = Element;
```

Comment:
Should we check if the Element is null? Or that doesn't make any difference?

Reply:
Element won't be null here. So, I believe, we can skip it (at least for now).